Add helm trustee install guide, add some useage details to other trustee install guides (#191)
confidential-containers/confidentialcontainers.org: content/en/docs/attestation/installation/_index.md

---

Trustee can be deployed in several different configurations.

> In every install scenario, Trustee should be deployed in a trusted environment.

Reviewer: ```suggestion
In every installation scenario, Trustee should be deployed in a trusted environment.
```

Author: thanks, updated!

This could be a local server, some trusted third party, or even another enclave.
Official support for deploying Trustee inside of Confidential Containers
is being developed.

## Choosing an installation method

| Method | Best for |
|--------|----------|

> | [Helm](helm/) | Delveopment or production deployments, where you want declarative configuration or advanced options (custom storage, BYOK, IBM SE) |

Reviewer: ```suggestion
| [Helm](helm/) | Development or production deployments, where you want declare configuration or advanced options (custom storage, BYOK, IBM SE) |
```

Author: thanks! updated

> | [Kubernetes Operator](kubernetes/) | Kubernetes deployments with operator-managed lifecycle  |

Reviewer 2: To match the title of that page:
```suggestion
| [Trustee Operator](kubernetes/) | Kubernetes deployments with operator-managed lifecycle  |
```

Author: updated, thanks!
